resources/assets/app.scss

```diff
@@ -52,3 +52,7 @@ button.delete {
   font-weight: bold;
   margin-right: 5px;
 }
+
+.next-page {
```

Comment:
I feel like, as much as possible, we should keep component-level styles in the components directory. Similar to [this](https://github.com/DoSomething/rogue/tree/master/resources/assets/components/StatusButton) . Otherwise we risk this app.scss file getting bloated.